godtools/App/Features/ToolsFilter/Presentation/ToolFilterCategorySelectionView/ToolFilterCategorySelectionViewModel.swift

```diff
@@ -25,14 +25,16 @@ class ToolFilterCategorySelectionViewModel: ObservableObject {
     
     let selectedCategory: CategoryFilterDomainModel
```

Comment:
Hey @rachaelblue I wasn't able to get a binding to work in what I had hoped for from ToolsViewModel to the SelectCategory and SelectLanguage ViewModels.  Let's stick with the approach of injecting the Subjects for now. 

The only change requests I'd suggest are updating both the selectedCategory and selectedLanguage to be Published properties and then update those when the category is tapped and language is tapped this way the View state will update.